privcount/util.py

```diff
@@ -728,3 +737,68 @@ def prob_exit(consensus_path, my_fingerprint, fingerprint_pool=None):
     sum_of_sq = sum_of_sq_bw/(TEWBW**2)
     return prob, sum_of_sq
 """
+
+class TrafficModel(object):
+
+    def __init__(self, states, start_p, trans_p, emit_p):
+        self.states = states
+        self.start_p = start_p
+        self.trans_p = trans_p
+        self.emit_p = emit_p
+
+    def get_counter_labels(self):
+        labels = []
+        for state in self.emit_p:
+            for direction in self.emit_p[state]:
+                labels.append("TrafficModelTotalEmissions_{}{}".format(state, direction))
```

Comment:
There should be an underscore here between state and direction, to avoid an (admittedly unlikely) collision if the end of one state matches the beginning of another direction in just the right way.